templates/sync-catalog-deployment.yaml

```diff
@@ -117,6 +123,16 @@ spec:
                 {{- if .Values.syncCatalog.addK8SNamespaceSuffix}}
                 -add-k8s-namespace-suffix \
                 {{- end}}
+                {{- if .Values.global.consulNamespacesEnabled }}
+                -enable-namespaces=true \
```

Comment:
Could this also be just `-enable-namespaces` since it's a boolean flag?

Reply:
It could, but I prefer to be explicit. This also follows the pattern set elsewhere in the Helm chart.